packages/contracts/contracts/QuadraticVoteCollectModule.sol

```diff
@@ -103,41 +76,28 @@ contract QuadraticVoteCollectModule is FeeModuleBase, FollowValidationModuleBase
     }
 
     function _processCollect(address collector, uint256 profileId, uint256 pubId, bytes calldata data) internal {
-        /// decode  data
         (address _currency, uint256 _amount) = abi.decode(data, (address, uint256));
-        address grantsRoundAddress = _dataByPublicationByProfile[profileId][pubId].grantsRoundAddress;
-        uint256 roundStartTime = IRoundImplementation(grantsRoundAddress).roundStartTime();
-        uint256 roundEndTime = IRoundImplementation(grantsRoundAddress).roundEndTime();
-        require(block.timestamp > roundStartTime && block.timestamp < roundEndTime, "Round is not in session");
 
-        //validate data
-        uint256 amount = _dataByPublicationByProfile[profileId][pubId].amount;
-        address currency = _dataByPublicationByProfile[profileId][pubId].currency;
-        _validateDataIsExpected(data, currency, amount);
-        //check signature validity
+        _validateDataIsExpected(data, _currency, _amount);
```

Comment:
see other comment on _validateDataisExpected on line 169

Reply:
Resolving this to keep discussion focussed